### PR TITLE
NO-ISSUE: Add additional operator user guide doc

### PR DESCRIPTION
**Pull request by akalenyu**
Common place for user-oriented-notes regarding additional AI operators.

Signed-off-by: Alex Kalenyuk <akalenyu@redhat.com>

## List all the issues related to this PR

- [ ] New Feature 
- [ ] Enhancement 
- [ ] Bug fix
- [ ] Tests
- [x] Documentation
- [ ] CI/CD 

## What environments does this code impact?

- [ ] Automation (CI, tools, etc)
- [ ] Cloud
- [ ] Operator Managed Deployments
- [x] None

## How was this code tested?

- [ ] assisted-test-infra environment
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [ ] Waiting for CI to do a full test run
- [ ] Manual (Elaborate on how it was tested)
- [x] No tests needed

## Assignees

/cc @avishayt 
/cc @

## Checklist

- [ ] Title and description added to both, commit and PR.
- [ ] Relevant issues have been associated (see [CONTRIBUTING] guide)
- [ ] Reviewers have been listed
- [ ] This change does not require a documentation update (docstring, `docs`, README, etc)
- [ ] Does this change include unit-tests (note that code changes require unit-tests)

## Reviewers Checklist

- Are the title and description (in both PR and commit) meaningful and clear?
- Is there a bug required (and linked) for this change?
- Should this PR be backported?

[Kubernetes community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: docs/user-guide/additional-operator-notes.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Additional OLM operator plugins
+
+## Notes
+
+### Local Storage Operator (LSO)
+
+### OpenShift Virtualization (CNV)
```
Comment (avishayt) [inline docs/user-guide/additional-operator-notes.md:7]:
```suggestion
## OpenShift Virtualization (CNV)
```

File: docs/user-guide/additional-operator-notes.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+- When deploying CNV on Single Node OpenShift (SNO), [hostpath-provisioner](https://github.com/kubevirt/hostpath-provisioner) (part of the CNV product) storage is automatically opted in and set up to use, to enable persisting VM disks.  
+This is done with the thought in mind that most virtualization use cases require persistence.  
+The hostpath-provisioner is set up to utilize an LSO PV as the backing storage for provisioning dynamic hostPath volumes on.
+
+### OpenShift Data Foundation (ODF)
```
Comment (avishayt) [inline docs/user-guide/additional-operator-notes.md:12]:
```suggestion
```

**Comment by avishayt** [comment]
@romfreiman please review

File: docs/user-guide/additional-operator-notes.md (hunk 1 above)
Comment (akalenyu) [reply docs/user-guide/additional-operator-notes.md]: done

**Review by avishayt — APPROVED** [review]

**Pull request merged**